Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -221,7 +221,7 @@ function IsPrBuild
 
 function GetSonarQubeBuildDirectory
 {
-    $agentBuildDirectory = GetTaskContextVariable "Agent.BuildDirectory" 
+    $agentBuildDirectory = $env:Agent_BuildDirectory 
```

Comment:
No, actually in the tasks there is a mix of both usages. I will ask for guidance.